lawnchair/res/values-v28/colors.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="system_accent1_100">@android:color/transparent</color>
```

Comment:
Please use something more appropriate than transparent. This can cause visual glitches or even crashes in some places.

Reply:
Let's leave it like this for now if you say it works